staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -43,14 +43,22 @@ var (
 		},
 		[]string{"operation", "type"},
 	)
-	objectCounts = compbasemetrics.NewGaugeVec(
+	etcdObjectCounts = compbasemetrics.NewGaugeVec(
 		&compbasemetrics.GaugeOpts{
 			Name:           "etcd_object_counts",
-			Help:           "Number of stored objects at the time of last check split by kind.",
+			Help:           "(Deprecated) Number of stored objects at the time of last check split by kind. This metric is replaced by apiserver_storage_object_counts.",
```

Comment:
Oh you don't actually need to include the 'Deprecated' part in the help text. 
You can use: `				DeprecatedVersion: "1.21.0",` in the GaugeOpts directly.

Reply:
Done